ingestion/src/metadata/ingestion/source/sample_data.py

```diff
@@ -339,10 +341,14 @@ def ingest_tables(self) -> Iterable[OMetaDatabaseAndTable]:
             ),
         )
         for table in self.tables["tables"]:
-            table_metadata = Table(**table)
-            table_and_db = OMetaDatabaseAndTable(table=table_metadata, database=db)
-            self.status.scanned("table", table_metadata.name.__root__)
-            yield table_and_db
+            resp = self.metadata.list_entities(entity=User, limit=5).json()
+            for sql_object in table["tableQueries"]:
+                id = json.loads(resp).get("entities")[random.choice(range(5))]["id"]
```

Comment:
To make it easier to handle, the `list_entities` method in the API returns an `EntityList` object. If you want only specific `User`s, you can run:

```python
res = self.metadata.list_entities(entity=User, limit=5)
users = res.entities  # This is now a `List[User]`
```

In the API everything should be properly typed for easy building and retrieval. If some castings are missing we can open an issue to get them fixed

Thanks!

Reply:
Link to [EntityList](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/ometa/ometa_api.py#L85)